pinot-tools/src/main/java/org/apache/pinot/tools/Quickstart.java

```diff
@@ -209,6 +212,35 @@ public void run() {
     printStatus(Color.GREEN, "You can always go to http://localhost:9000/query/ to play around in the query console");
   }
 
+  protected void waitForAllDocsLoaded(QuickstartRunner runner, long timeoutMs)
+      throws Exception {
+    final long countStarResult = 97889L;
+    TestUtils.waitForCondition(new Function<Void, Boolean>() {
```

Comment:
TestUtils? better to use RetryPolicies in PinotCommon. It allows you to try every 5 seconds until 60 seconds.

Reply:
Changed to `RetryPolicies`.